src/docs/concepts-development/content-management/index.md

```diff
@@ -0,0 +1,133 @@
+have several **blocks**. Depending on which kinds of sections and blocks
+you choose for your page, you can create a wide range of content. In the
+future, it will also be possible to create custom sections.
+
+TODO: IS THE LEVEL OF TECHNICALITY OK FOR A FLEX DOCS ARTICLE CF
+OPERATOR GUIDE?
+
+## Flex Content modeling glossary
+
+_TODO: A better place for this section?_
+
+Before diving deeper into the different content model types in Flex,
+here is a short glossary of the most central content related terms.
+
+**Page**: The collection of different content elements under a specific
+URL. Pages can have fields, sections, or blocks within them. A page can
+be fetched as [an asset](/references/assets) from the Flex Asset
+Delivery API. Example: `landing-page`.
+
+**Section**: The main element of a page. In Flex, each section has a
+specific content model that determines the information that the section
+must have to function properly. Sections can have fields and blocks
+within them. In Flex, sections are the main way to distinguish pages
+from each other. Example: article, feature, columns.
+
+**Block**: Blocks are optional, but they are very useful for more
+extensive pieces of content. Different section content models can show
+blocks in different ways, depending on the design. Blocks can have
+fields within them. Example: default block.
+
+**Field**: The simplest content element in Flex. Conveys a single piece
+of information. Example: text field, button, image.
+
+![Example page with levels of content modeling elements](./content_modeling.png)
+
+## Sections
+
+Flex has X default content sections that you can select for your page.
+All sections have a title, an ingress, and a button for a call to
+action.
+
+### Article
+
+Article sections are great for prose text. If your content is longer,
+you can add blocks with different levels of headings and text
+paragraphs. The blocks are displayed in a column one after the other, as
+if on a blog page.
+
+### Columns
+
+Columns sections divide the blocks within the section into pre-defined
+columns, and the operator can decide the number of columns between 1-4.
+The columns pattern is great for featuring specific options in parallel,
+whether they are benefits, testimonials, or featured locations.
+
+### Features
+
+Features sections, by default, show blocks of visual and text content so
+that the block contents are on a single row, alternating the order, and
+different blocks are in a column. As with all content sections, the
+designer of your website can make changes as to how exactly the blocks
+get displayed on the page.
+
+## Blocks
+
+Currently, Flex has a single type of block that you can add to any
+section. The block contains a title, and you can determine the level of
+the title (page title, section title, section subtitle). A block can
+also have text content and a button.
+
+Block text content can be modified with Markdown. This allows the
```

Comment:
Should we have some examples of how markdown is edited or should that be in operator-guides

Reply:
I think we could add a link to Markdown documentation on this, and then have more examples in the operator guide, ping @thomas-rocca .

Reply:
Thanks! Link added to article.